FeedReader.Tests/FeedReaderTest.cs

```diff
@@ -173,7 +173,9 @@ public void TestReadNoticiasCatolicas()
             Assert.IsTrue(feed.Items.Count > 0);
         }
 
+        // The timedoctor feed does not exist anymore
         [TestMethod]
+        [Ignore]
```

Comment:
the url has changed to: https://www.timedoctor.com/blog/feed/
please replace the ignore attribute with the new url.